src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -316,6 +316,10 @@ input.inline {
     margin-left: 7px;
 }
 
+.margin-left-3px {
+    margin-left: 3px !important;
```

Comment:
From [Teammates CSS coding standards](https://docs.google.com/document/d/1wA9paRA9cS7ByStGbhRRUZLEzEzimrNQjIDPVqy1ScI/pub)

<img width="767" alt="screen shot 2018-06-19 at 1 39 59 pm" src="https://user-images.githubusercontent.com/12574756/41578768-6ee1a0fe-73c6-11e8-9152-6483dd213d8e.png">

Please add a comment about the `.margin-left-3px {` to give reason for adding `!important`

Reply:
I moved `.margin-left-3px` before `.margin-left-7px` as you suggested.

I also added the comment explaining the need to use `!important`, similar to the way it is done elsewhere (line 164 of `teammatesCommon.css`,  `.mouse-hover-only`